syntax = "proto3";

package hello;
option go_package = "main";

service HelloService {
  rpc Hello(HelloRequest) returns (HelloReply){}
}

message HelloRequest { string message=1;}
message HelloReply { string message=1;}
